lib/twilio_api.rb

```diff
@@ -0,0 +1,21 @@
+require 'twilio-ruby'
+require 'dotenv'
+Dotenv.load
+
+module Messenger
+
+def sendMessage(message_out,receiver)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `receiver`. If it's necessary, use `_` or `_receiver` as an argument name to indicate that it won't be used.<br>Style/MethodName: Use snake_case for method names.